client/cypress/support/po.ts

```diff
@@ -0,0 +1,177 @@
+export class EditorViewPageObject {
```

Comment:
I guess, once we introduce more PageObjects, we can move those into dedicated files.